src/numerics/CVodesIntegrator.cpp

```diff
@@ -74,6 +73,29 @@ extern "C" {
         integrator->m_error_message = msg;
         integrator->m_error_message += "\n";
     }
+    
+    #if CT_SUNDIALS_VERSION>=30 //These functions only work with Sundials 3.0 or newer
```

Comment:
What happens if `CT_SUNDIALS_VERSION` is less than 3.0? Does the code fail with a useful error message?

Reply:
This was updated to pass CI. It throws a `CanteraError` now.